README.md

```diff
@@ -1,4 +1,6 @@
  # Colonial Marines ALPHA
- [![Build Status](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018.svg?branch=master)](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018)
+ [![Build Status](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018.svg?branch=master)](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018) [![Krihelimeter](https://www.krihelinator.xyz/badge/ColonialMarines-Mirror/ColonialMarines-2018)](https://www.krihelinator.xyz)  
+ [![forthebadge](https://forthebadge.com/images/badges/contains-technical-debt.svg)](https://forthebadge.com) [![forinfinityandbyond](https://user-images.githubusercontent.com/5211576/29499758-4efff304-85e6-11e7-8267-62919c3688a9.gif)](https://www.reddit.com/r/SS13/comments/5oplxp/what_is_the_main_problem_with_byond_as_an_engine/dclbu1a) [![madewithdmca](https://img.shields.io/badge/made--with-D--M--C--A-blue.svg?longCache=true&style=for-the-badge)](https://shields.io)
```

Comment:
Replace shields.io in the last part with this video https://www.youtube.com/watch?v=qvM6Mg0VtO0